canonical-web-and-design/maas-ui
# feat(ui): add notifications to machine storage tab

## Done

- Added notifications to the storage tab based on the angular version. The only thing I've changed is showing the commissioning error by itself instead of showing every possible error notification, which felt like overkill.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to a few legacy machine storage tabs and check that the notifications match the new react version, except for machines that have not yet been commissioned in which case where there should only be one notification.

## Fixes

Fixes canonical-web-and-design/MAAS-squad#2155

## Screenshot
![Screenshot_2020-11-10 illegal-finch maas storage bolla MAAS(1)](https://user-images.githubusercontent.com/25733845/98636522-c26b7c00-2372-11eb-8321-3069bb7da52f.png)
